registry-image-sync/readme.md

```diff
@@ -0,0 +1,25 @@
+# Overview
+
+This action helps to sync images between two container registries
+
+# Example
+
+on:
+  workflow_dispatch:
+```yaml
+jobs:
+  cleanup:
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/skopeo/stable
+    steps:
+      - name: Sync images
+        uses: scality/actions/registry-image-sync
+        with:
+          SOURCE_REPO: <repo-name> # Ex bert-e
+          IMAGE_NAME: <image-name> # Ex bert-e
```

Comment:
You're making some parameters as required but not mentioning them here, feel free to setup default values for the destination.